ansible/group_vars/all

```diff
@@ -435,8 +442,9 @@ metrics:
 
 user_events: "{{ user_events_enabled | default(false) | lower }}"
 
-durationChecker:
-    timeWindow: "{{ duration_checker_time_window | default('1 d') }}"
+zerodowntimeDeployment:
```

Comment:
This configuration is used to deploy all components without any downtime in downstream.
Since there are no corresponding ansible steps on the controller and invoker sides, it would not work but I kept it as is.
We can add the required changes in the future.

Reply:
```suggestion
zeroDowntimeDeployment:
```

Reply:
How complicated are the required changes? And how exactly does the zero downtime work?

Reply:
I just want to focus on running the new scheduler in this PR.
I believe it's not that difficult.

The procedure is like

1. Disable some invokers.
2. Check if any activations running on them.
3. If yes, wait until they are complete. If not, redeploy invokers.
4. Disable some schedulers.
5. Check if any activations are in the schedulers.
6. If yes, wait until they are forwarded to other schedulers. If not, redeploy schedulers.
7. Remove some controllers from Nginx.
8. Disable controllers.
9. Wait until all activations finish.
10. Redeploy controllers.
11. Repeat 1~10 steps until most of the components are redeployed.
12. For the last controller, scheduler, invoker, change the order to controller -> scheduler -> invoker.